src/Login.php

```diff
@@ -49,7 +53,7 @@ public static function register_mutation() {
 				],
 				'mutateAndGetPayload' => function( $input, AppContext $context, ResolveInfo $info ) {
 					// Login the user in and get an authToken and user in response.
-					return Auth::login_and_get_token( sanitize_user( $input['username'] ), trim( $input['password'] ) );
+					return Auth::login_and_get_token( sanitize_user( $input['username'] ), trim( $input['password'] ), $input['gRecaptchaResponse'] );
```

Comment:
@GautheyValentin There needs to be validation of some kind for `$input['gRecaptchaResponse']` seeing as it's not a required field it could be null.